annotation/app/tasks/resources.py

```diff
@@ -920,7 +920,11 @@ def finish_task(
 
             # TODO extensive coverage, annotators, manifest_url
             extensive_coverage = task.jobs.extensive_coverage
-            if extensive_coverage and extensive_coverage > 1:
+            if (
+                extensive_coverage
+                and extensive_coverage > 1
+                and AGREEMENT_SCORE_ENABLED == "true"
```

Comment:
it's up to you, but this way seems more Pythonic to me:

#.env
AGREEMENT_SCORE_ENABLED=0

#code.py
if AGREEMENT_SCORE_ENABLED:
    ....

Reply:
I think this is less explicit